domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -167,6 +169,7 @@ public String toString() {
     }
     spaceAppend(sb, limit.toString());
     spaceAppend(sb, offset.toString());
+    log.info("CQL >>> SQL " + this.query + " >>>" + sb.toString());
     return sb.toString();
   }
```

Comment:
This invokes sb.toString() two times:

    log.info("CQL >>> SQL: " + this.query + " >>>" + sb.toString());
    return sb.toString();

Can it be wrapped?

    String sql = sb.toString();
    if (log.isInfoEnabled()) {
      log.info("CQL >>> SQL: " + this.query + " >>>" + sql);
    }
    return sql;